src/textual/_styles_cache.py

```diff
@@ -66,6 +66,11 @@ class StylesCache:
 
     """
 
+    _border_title: str = ""
```

Comment:
I don't think we will need these, since we can inspect the widget object when we come to render it.